src/compiler/scala/tools/nsc/ast/parser/Parsers.scala

```diff
@@ -994,10 +1010,7 @@ self =>
     def finishBinaryOp(isExpr: Boolean, opinfo: OpInfo, rhs: Tree): Tree = {
       import opinfo._
       if (targs.nonEmpty)
-        if (currentRun.isScala3)
-          syntaxError(offset, "type application is not allowed for infix operators")
-        else
-          deprecationWarning(offset, "type application will be disallowed for infix operators", "2.13.11")
+        migrationWarning(offset, "type application is not allowed for infix operators", "2.13.11")
```

Comment:
intentially demoted?

Reply:
There were a few where I wasn't sure and went for uniformity, but not because of any inner resolve or dogma.